kroxylicious-operator/pom.xml

```diff
@@ -322,4 +322,143 @@
+                            </execution>
+                            <execution>
+                                <!-- load the container image from the temp file into minikube -->
+                                <id>minikube-load</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <workingDirectory>..</workingDirectory>
+                                    <executable>minikube</executable>
+                                    <arguments>
+                                        <argument>image</argument>
+                                        <argument>load</argument>
+                                        <argument>${image.archive.file}</argument>
+                                    </arguments>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <!-- copy from packaging into target/packaged, interpolating $[ ] (e.g. to use the container image) -->
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <version>3.3.1</version>
+                        <executions>
+                            <execution>
+                                <id>copy-operator-install</id>
+                                <goals><goal>copy-resources</goal></goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <delimiters>
+                                        <delimiter>$[*]</delimiter>
+                                    </delimiters>
+                                    <resources>
+                                        <resource>
+                                            <directory>packaging</directory>
+                                            <filtering>true</filtering>
+                                        </resource>
+                                    </resources>
+                                    <outputDirectory>target/packaged</outputDirectory>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>copy-crds</id>
+                                <goals><goal>copy-resources</goal></goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <resources>
+                                        <resource>
+                                            <directory>../kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8</directory>
```

Comment:
One thing I'm still not doing in this PR is trying to change the filenames of the CRD YAMLs to match those of the other manifests from the `install` directory. I think we're now on a version of JOSDK where it's less opinionated about there those live, so it ought to be possible.